src/client/request_closure.cpp

```diff
@@ -31,8 +32,8 @@ namespace client {
 
 void RequestClosure::Run() {
     ReleaseInflightRPCToken();
-    if (suspendRPC_) {
-        MetricHelper::DecremIOSuspendNum(metric_);
+    if (CURVE_UNLIKELY(slowRequest_)) {
```

Comment:
Does the slow requests here means inflight slow requests? It will be cleared when finished.

Reply:
Yes, in addition to the previous error log, there is an extra metric